src/Uno.UI/UI/Xaml/FontFamilyHelper.cs

```diff
@@ -5,36 +5,10 @@
 
 namespace Windows.UI.Xaml;
 
-public partial class FontFamilyHelper
+internal static partial class FontFamilyHelper
 {
-#if NETFX_CORE
-	public static Windows.UI.Xaml.Media.FontFamily Create(string familyName)
```

Comment:
#8339 doesn't mention removing the `Create` API, but it felt good to do. The API shouldn't be needed